e2e/testdata/fn-eval/resource-deletion/pkg/.expected/diff.patch

```diff
@@ -0,0 +1,32 @@
+diff --git a/resources.yaml b/resources.yaml
+index 10cd18b..0848ba0 100644
+--- a/resources.yaml
++++ b/resources.yaml
+@@ -1,27 +1,6 @@
+-# Copyright 2021 Google LLC
+-#
+-# Licensed under the Apache License, Version 2.0 (the "License");
+-# you may not use this file except in compliance with the License.
+-# You may obtain a copy of the License at
+-#
+-#      http://www.apache.org/licenses/LICENSE-2.0
+-#
+-# Unless required by applicable law or agreed to in writing, software
+-# distributed under the License is distributed on an "AS IS" BASIS,
+-# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+-# See the License for the specific language governing permissions and
+-# limitations under the License.
+-
```

Comment:
@mengqi Looks like the starlark function will remove the comments.

Reply:
Since starlark function is updated now to handle this, so this will be updated now.